src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -823,6 +825,19 @@ const EVENT_LINKS: EventLink[] = [
       );
     },
   },
+  {
+    linkRelation: TIME_OF_NEED_HEALING,
+    linkingEventId: TALENTS_EVOKER.TIME_OF_NEED_SUMMON.id,
+    linkingEventType: EventType.Summon,
+    referencedEventId: [SPELLS.VERDANT_EMBRACE_HEAL.id, SPELLS.TIME_OF_NEED_LIVING_FLAME.id],
+    referencedEventType: EventType.Heal,
+    forwardBufferMs: 8000, //8 seconds, the duration of Time of Need
```

Comment:
Put this into a constant at the top of the file similar to other links, makes it a lot easier to refactor stuff on patches.